GOVERNANCE.md

```diff
@@ -25,4 +26,25 @@ Contributors are encouraged to ask for collaborator access once you have had a f
 
 ## Process for releasing a new version of Wikimate
 
-[TODO]
+A PR should be created with all relevant changes to update the repository for the upcoming release.
+(See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
+It should apply the following actions:
+
+1. Change the "Unreleased" heading in the CHANGELOG.md file
```

Comment:
I can't recall how detailed the previous discussion was (I did a search but couldn't find it), but the rationale here is that if we're claiming/recommending [Keep a Changelog](http://keepachangelog.com/) compliance, then we should use the heading they propose, i.e. "Unreleased" (which IMO is also more explicit than "Current development", and fits better with the other headings), unless there are strong arguments to deviate from it.